daemon/snap.go

```diff
@@ -154,10 +155,10 @@ func allLocalSnapInfos(st *state.State, all bool) ([]aboutSnap, error) {
 
 // appJSON contains the json for snap.AppInfo
 type appJSON struct {
-	Name         string   `json:"name"`
-	Daemon       string   `json:"daemon"`
-	DesktopFiles []string `json:"desktop-files"`
-	Aliases      []string `json:"aliases,omitempty"`
+	Name        string   `json:"name"`
+	Daemon      string   `json:"daemon"`
+	DesktopFile string   `json:"desktop-files,omitempty"`
```

Comment:
Indeed, we could simply ask gnome-software to implement the lookup. The upside of having it in the API is that if we ever change our mind about the convention there will be less software to change (also that we do not need to implement the convention in each software that uses snapd (snapweb, the kde version of gnome-software etc)). However given that its a really small piece of convention its not a very strong argument :) Happy to remove it from the API again.